webapp/team.py

```diff
@@ -33,35 +70,49 @@ def get_all_teams():
     ]
 
 
-def get_all_mattermost_handles():
-    lp = Launchpad.login_anonymously(
-        "webteam.canonical.com", "production", ".", version="devel"
-    )
-    team = lp.people["canonical-webmonkeys"]
-
-    return [person.name for person in team.members]
-
-
 @webteam.route("/")
 def index():
-    display_names = get_all_display_names()
-    display_names.sort()
+    all_members_names = [
+        member["name"] for member in get_team_members("canonical-webmonkeys")
+    ]
+    if request.headers.get(
+        "Content-Type"
+    ) and "application/json" in request.headers.get("Content-Type"):
+        return jsonify(all_members_names)
+    else:
+        return render_template(
+            "team/index.html", display_names=all_members_names
+        )
+
+
+@webteam.route("/mattermost")
+def mattermost():
+    mattermost_handles = [
+        person["username"]
+        for person in get_team_members("canonical-webmonkeys")
+    ]
 
     if request.headers.get(
         "Content-Type"
     ) and "application/json" in request.headers.get("Content-Type"):
-        return jsonify(display_names)
+        return jsonify(mattermost_handles)
     else:
-        return render_template("team/index.html", display_names=display_names)
+        return render_template(
+            "team/mattermost.html", mattermost_handles=mattermost_handles
+        )
 
 
 @webteam.route("/random")
 def random():
-    display_names = get_all_display_names()
+    all_members = get_team_members("canonical-webmonkeys")
```

Comment:
There are a few non-webteam members here like Nick and Ulas. Not sure it matters but they will likely return as designers.

Reply:
I see, This is indeed not ideal but should help the designers team to have less refresh to do.
If the list of non-designers members increases we should create a separate LP team for the designers team as well similarly to the web team engineers.